tutorials/lab/rabbit.rst

```diff
@@ -0,0 +1,128 @@
+.. _rabbit:
+
+=================
+Flux with Rabbits
+=================
+
+
+How to Allocate Rabbit Storage
+------------------------------
+
+Request rabbit storage allocations for a job
+by setting the ``.attributes.system.dw`` field in a jobspec to
+a string containing one or more DW directives, or to a list of
+singleton DW directives.
+
+DW directives are strings that start with ``#DW``. Directives
+that begin with ``#DW jobdw`` are for requesting storage that
+lasts the lifetime of the associated flux job. Directives that
+begin with ``#DW copy_in`` and ``#DW copy_out`` are for
+describing data movement to and from the rabbits, respectively.
+
+The usage is most easily understood by example.
+
+
+Examples of jobdw directives
+----------------------------
+
+Requesting a 10 gigabyte XFS file system on the
+command line:
+
+.. code-block:: console
+
+	$ flux alloc -N2 --setattr=dw="#DW jobdw type=xfs capacity=10GiB name=project1"
+
+Requesting two XFS file systems in a batch script:
+
+.. code-block:: bash
+
+	#!/bin/bash
+
+	#FLUX: -N 2
+	#FLUX: -q pdebug
+	#FLUX: --setattr=dw="""
+	#FLUX: #DW jobdw type=xfs capacity=1TiB name=my_xfs
+	#FLUX: #DW jobdw type=lustre capacity=10GiB name=my_lustre
+	#FLUX: """
+
+	echo "Hello World!" > $DW_JOB_my_lustre/world.txt
```

Comment:
Shouldn't this line be dropped?

Reply:
How come? I thought it couldn't hurt to show some extremely basic usage.

Reply:
I'm not very familiar with Flux job scripts, but won't this run on the node submitting the job?  In which case `$DW_JOB_my_lustre/` won't be defined nor will the filesystem be available.

Reply:
No it should work just like a Slurm job script, so it will run in the job allocation.

Reply:
Ah okay, thanks for clearing that up for me.